src/map/clif.cpp

```diff
@@ -17674,6 +17674,10 @@ void clif_parse_configuration( int32 fd, map_session_data* sd ){
 
 			sd->hd->homunculus.autofeed = flag;
 			break;
+		case CONFIG_DISABLE_SHOWCOSTUMES:
+			sd->status.disable_showcostumes = !flag;
+			pc_set_costume_view(sd);
+			break;
 		default:
```

Comment:
We dont need to send the configuration in clif_parse_LoadEndAck like the others?

Reply:
I dont know what is the purpose of the clif_parse_LoadEndAck